code/scripting/api/libs/hud.cpp

```diff
@@ -163,19 +204,26 @@ ADE_FUNC(setHUDGaugeColorInMission, l_HUD,
 
 ADE_FUNC(getHUDGaugeColorInMission,
 	l_HUD,
-	"number|string gaugeNameOrIndex",
+	"number|string gaugeNameOrIndex, [boolean ReturnType]",
```

Comment:
Same comment as `gr.getColor` about true vs. false.  Also the documentation should indicate what true means and what false means.